src/NServiceBus.NHibernate/FodyWeavers.xsd

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Delete this file.

Reply:
I constantly fall for this one. Thank you.

Reply:
Done